### PR TITLE
Support Django >= 1.10

### PR DESCRIPTION
**Pull request by ivlevdenis**
Fix get_field_by_name, get_all_related_objects, get_all_related_many_to_many_objects for Django >= 1.10

### CODE REVIEW

File: dynamic_rest/meta.py [hunk 1]
```diff
@@ -36,15 +39,30 @@ def get_model_field(model, field_name):
...
+            field = meta.get_field(field_name)
+        else:
+            field = meta.get_field_by_name(field_name)[0]
+        return field
     except:
+        if DJANGO110:
+            related_objs = (
+                f for f in meta.get_fields()
+                if (f.one_to_many or f.one_to_one)
+                   and f.auto_created and not f.concrete
```
Comment (aleontiev) [inline dynamic_rest/meta.py]: Unit tests look good, but there seems to be a minor PEP8 style violation on this line:

```
dynamic_rest/meta.py:52:20: E131 continuation line unaligned for hanging indent
```

Comment (ivlevdenis) [inline dynamic_rest/meta.py]: Thnx. Fixed.

**Comment by aleontiev** [comment]
@ivlevdenis these changes are awesome, thanks for working on them! I haven't worked with 1.10 yet but I'm excited to try it out.

It looks like there are some merge conflicts, most likely with this recently merged PR that added DRF 3.4 compatibility: https://github.com/AltSchool/dynamic-rest/pull/120.

Please pull in the changes from #120 and update the compatibility table in README.md to reflect support for 1.10.

**Comment by aleontiev** [comment]
Looks great!

**Pull request merged**